_data/sidebars/sidebar_ioos.yml

```diff
@@ -63,11 +63,15 @@ entries:
       - title: Global Attributes
         url: /ngdac-netcdf-file-format-version-2.html#global-attributes
         output: web
-        # subfolderitems:
-        # - title: Caveats
-        #   url: /ngdac-netcdf-file-format-version-2.html#global-attributes
-        #   output: web
+        subsubfolders:
```

Comment:
I'm not a big fan of the third level of TOC. Too many clicks. But I'll defer to you if you think this is best